src/cljs/main/broadfcui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -86,27 +98,31 @@
              (list
               [comps/SidebarButton {:color :success-state
                                     :text "Save" :icon :done
+                                    :disabled? (when redacted? "Choose an available snapshot")
                                     :data-test-id (config/when-debug "save-editted-method-config-button")
                                     :onClick #(parent :-commit)}]
               [comps/SidebarButton {:color :exception-state :margin :top
                                     :text "Cancel Editing" :icon :cancel
                                     :data-test-id (config/when-debug "cancel-edit-method-config-button")
                                     :onClick #(parent :-cancel-editing)}])
-             (concat
+             (list
```

Comment:
Is there a reason for this change?

Reply:
It changed from starting with a vector to starting with a single item. I'd really like to develop a better way of handling things like this--maybe something like `flatten` but it can tell that `[:div {} stuff]` is a DOM node and shouldn't be flattened.